src/site/navigation/sidebar_nav.drupal.liquid

```diff
@@ -1,110 +1,113 @@
-<nav id="va-detailpage-sidebar" data-drupal-sidebar="true" class="va-drupal-sidebarnav usa-width-one-fourth va-sidebarnav">
-    <div>
+{% if buildtype != 'vagovprod' %}
```

Comment:
https://github.com/department-of-veterans-affairs/vets-website/pull/10881/files#r333994696